code/game/objects/items/devices/flash.dm

```diff
@@ -262,15 +264,14 @@
 
 /obj/item/flash/memorizer
 	name = "memorizer"
-	desc = "If you see this, you're not likely to remember it any time soon."
+	desc = "If you see this, you're not likely to remember it any time soon." // Why doesn't this at least delete your notes smh
 	icon_state = "memorizer"
 	item_state = "nullrod"
 
 /obj/item/flash/armimplant
 	name = "photon projector"
 	desc = "A high-powered photon projector implant normally used for lighting purposes, but also doubles as a flashbulb weapon. Self-repair protocols fix the flashbulb if it ever burns out."
-	cooldown_duration = 2 SECONDS
```

Comment:
ERT balance change? 🤨 (add to changelog please)

Reply:
Added back - forgot about that!